internal_ws/yksg/src/lib.rs

```diff
@@ -1,6 +1,9 @@
-//! The stopgap interpreter which is invoked during a guard failure. It picks up right after the
-//! guard and interprets SIR getting us safely back to a control point in the the meta-tracer from
-//! which point normal interpretation can continue.
+//! The stopgap interpreter.
+//!
+//! This is a SIR interpreter which is invoked after a guard failure. It interprets from the point
+//! of divergence in the compiled trace up until the next control point. Once arrived at the
```

Comment:
Oh, it might help to define "compiled" here...

Reply:
The current comment is a bit hard to parse. I think you can simplify it to something like "After a guard failure, the StopgapInterpreter takes over execution. It interprets SIR to execute the program from the guard failure until execution arrives back to the control point, at which point the normal interpreter can continue."

Reply:
Fixed in 55137d0cb99e3c6dabb17720a353398f600e196e.